### ISSUE TITLE
Feature Request - Working Printer in cockpit?

### ISSUE THREAD

**Issue by Thilo277** [legoboyvdlp/A320-family#269]
Would it be possible to implement a working printer into the cockpit which can be activated, like in real life via the MCDU, for example for printing TAF/METAR etc., like it is in the E-Jet Family by nl256?

**Comment by merspieler** [comment]
certainly a feature to have.
I can't find nl256... do you mean the version be [tdammers](https://github.com/tdammers/E-jet-family-YV)?

**Comment by Octal450** [comment]
nl256 = tdammers

Kind Regards,
Josh

**Comment by merspieler** [comment]
okay, thanks josh

**Comment by Thilo277** [comment]
Yeah, sorry for confusion, wrote his discord name, not his real/github name by accident
